### ISSUE TITLE
Links and buttons not in the correct tab order in the survey page

### ISSUE THREAD

**Issue by Ifycode** [ushahidi/platform#4150]
Based on the fix I made here in [PR 259](https://github.com/ushahidi/platform-pattern-library/pull/259) that should affect all buttons on the site, button elements that should have been skipped now receive focus - this is the reason why the buttons are not skipped in the screenshot below.

**Describe the bug:**

The bug is noticeable once skipped elements have been fixed. When navigating through the page, buttons and links on the page do not have the correct tab-order. The one on the right receive focus before the one on the left, when it should have been from left to right.

**Screenshot:**

![Surveys - Obiagba Mary](https://user-images.githubusercontent.com/45185388/97820715-4c755f80-1c64-11eb-8b8a-6cb9d6b9e5d4.gif)

**Where was the bug observed:**

http://localhost:3000/settings/surveys

**Steps to reproduce the behavior:**
1. Login to your account.
2. Go to setting > surveys.
3. Navigate the page with the tab key.

Aha! Link: https://ushahiditeam.aha.io/features/PROD-894

**Comment by Ifycode** [comment]
Hi @Angamanga @rowasc, can I work on this?

**Comment by Angamanga** [comment]
@Ifycode Yes absolutely :)

**Comment by Ifycode** [comment]
Yay! Ok thank you @Angamanga
